src/test/java/io/mosip/mimoto/util/CredentialIssuerWellknownResponseValidatorTest.java

```diff
@@ -0,0 +1,233 @@
+package io.mosip.mimoto.util;
+
+import io.mosip.mimoto.dto.BackgroundImageDTO;
+import io.mosip.mimoto.dto.mimoto.CredentialDefinitionResponseDto;
+import io.mosip.mimoto.dto.mimoto.CredentialIssuerWellKnownResponse;
+import io.mosip.mimoto.dto.mimoto.CredentialSupportedDisplayResponse;
+import io.mosip.mimoto.dto.mimoto.CredentialsSupportedResponse;
+import io.mosip.mimoto.exception.ApiNotAccessibleException;
+import io.mosip.mimoto.exception.InvalidWellknownResponseException;
+import jakarta.validation.Validation;
+import jakarta.validation.Validator;
+import jakarta.validation.ValidatorFactory;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static io.mosip.mimoto.util.TestUtilities.getCredentialIssuerWellKnownResponseDto;
+import static io.mosip.mimoto.util.TestUtilities.getCredentialSupportedResponse;
+import static org.junit.Assert.assertThrows;
+
+public class CredentialIssuerWellknownResponseValidatorTest {
+
+    @Autowired
+    private Validator validator;
+
+    @BeforeEach
+    void setUp() {
+        ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+        validator = factory.getValidator();
+    }
+
+    @Test
+    public void shouldThrowExceptionWhenCredentialIssuerIsMissingInCredentialIssuerWellknownResponse() throws ApiNotAccessibleException {
```

Comment:
can we follow the AAA rule in the test, and have clear separation between all A's

and if we are expecting throws 
mention it like this in the annotation

@Test(expected = Exception.class)

this way its more clear

Reply:
with @test(expected) annotation, we can't assert on error messages sent along

Reply:
@vijay151096 before JUnit5, we used to assert exception with annotation. But with latest one, we can assert error code, messages using assertThrows method.